tests/test.py

```diff
@@ -36,6 +36,9 @@
     'NodeFirefoxDebug': 'node-firefox-debug',
     'StandaloneFirefox': 'standalone-firefox',
     'StandaloneFirefoxDebug': 'standalone-firefox-debug',
+
```

Comment:
Can you please add tests for the Standalone mode as well?

Reply:
Added.